dvc/command/init.py

```diff
@@ -53,7 +53,7 @@ def add_parser(subparsers, parent_parser):
         action="store_true",
         default=False,
         help=(
-            "Overwrite existing '.dvc' directory. "
+            "Overwrite existing '.dvc/' directory. "
```

Comment:
Hm, why do we need `/` here? It clarifies that this is a directory by explicitly saying that.

Reply:
Ah yes, sorry. This is not really about ' or `, I should've explained!

We do this all over the docs (14 instances) to be extra explicit about `.dvc/` dirs vs. `.dvc` DVC-files, even if the text mentions it's a dir, so I'm just used to doing this.

Should I revert it, or would you like to stick to same practice we use in docs?

Reply:
@jorgeorpinel oh, good point! Yeah, let's keep it as is. Thanks! :)

Reply:
Thank you!